examples/aws/example-build/envs.sh

```diff
@@ -27,4 +27,7 @@ export TF_VAR_root_dns_zone_id=<aws route53 zone id>
 export TF_VAR_subnet_bastion_0=<bastion subnet cidr 1>
 export TF_VAR_subnet_bastion_1=<bastion subnet cidr 2>
 
+# Logging bucket expiration days
+export TF_VAR_expiration_days=<expiration days>
```

Comment:
Do we really need this here, we have a lot of variables and if we put all of them here it does not really makes sense to me. Default 365 days is fine for me.